.github/workflows/dtypes_benchmark.yml

```diff
@@ -39,6 +39,8 @@ jobs:
         run: |
           invoke benchmark-dtypes
 
+        continue-on-error: true
```

Comment:
Should we add a `timeout-minutes` to the `run_dtypes_benchmark` job?

Reply:
We can set it up to `20` minutes. This are the actual times for it:
<img width="670" alt="image" src="https://github.com/user-attachments/assets/33efb615-dd3f-4d21-89b3-d16713e6b890">

Reply:
Done in https://github.com/sdv-dev/SDV/pull/2248/commits/36f19b338a8be0ccf6e9398fc174496dd6ce127e